frontend/src/metabase/query_builder/components/view/sidebars/QuestionDetailsSidebar.jsx

```diff
@@ -0,0 +1,28 @@
+import React, { useState } from "react";
+import PropTypes from "prop-types";
+
+import QuestionDetailsSidebarPanel from "metabase/query_builder/components/view/sidebars/QuestionDetailsSidebarPanel";
+import { SIDEBAR_VIEWS } from "./constants";
+
+QuestionDetailsSidebar.propTypes = {
+  question: PropTypes.object.isRequired,
+  onOpenModal: PropTypes.func.isRequired,
+};
+
+function QuestionDetailsSidebar({ question, onOpenModal }) {
+  const [view, setView] = useState(view);
```

Comment:
This component and `view` logic are obviously kind of pointless right now because there's only one possible view, but I'm keeping it because we will definitely be needing it with the return of BUCM.